autobot/api.py

```diff
@@ -8,77 +8,73 @@
 
 """Autobot API."""
 
+import copy
+
 import yaml
 
 from autobot.config_loader import Config
 from autobot.github import GitHubAPI
 
 
 class BotAPI:
-    """Generates report."""
+    """Bot's API."""
 
-    @classmethod
-    def __init__(cls, config: Config):
+    def __init__(self, config: Config):
         """Bot initialization."""
-        cls.config = config.config
-        cls.report = GitHubAPI(
-            cls.config["AUTOBOT_OWNER"], cls.config["AUTOBOT_GH_TOKEN"]
-        ).report(cls.load_repositories())
+        self.config = config
 
     @classmethod
-    def generate_report(cls, maintainer: str) -> dict:
-        """Generates a report for a maintainer."""
-        res = cls.report[0]["repos"]
-        print(res)
-        print(
-            "---------------------------------------------------------------------------------"
-        )
-        return res
+    def md_report(cls, report):
+        """Returns the report in markdown format."""
+        lines = []
+        for repo in report.keys():
+            repo_report = report[repo]
+            lines.append(
+                f"\n### [{repo_report['url'].split('/')[-1]}]({repo_report['url']})"
+            )
+            for (action, targets) in repo_report["actions"].items():
```

Comment:
```suggestion
            for action, targets in repo_report["actions"].items():
```

Parentheses are not needed to unpack the tuple (unless you had multiple nested tuples)